### ISSUE TITLE
11.1.3.1x Neuer Prüfschritt für nicht programmatisch exponierte Inhalte

### ISSUE THREAD

**Issue by detlevhfischer** [BIK-BITV/BIK-App-Test#24]
In einem laufenden App-Test habe ich das Problem, dass sich an vielen Stellen wichtige Textinhalte (Überschriften, Hinweise usw.) **gar nicht** über den Screenreader erreichen lassen. Sie sind irgendwie auf `inert` gestellt (wie `aria-hidden` im Web). Dies fällt am ehesten unter 1.3.1 Informationen und Beziehungen - es gibt da etwas Visuelles, das programmatisch nicht ermittelbar ist. Es passt aber in keinen der vorhandenen X.1.3.1 Prüfschritte.

Ich würde deshalb einen neuen Prüfschritt 11.1.3.1x "Inhalte programmatisch ermittelbar" vorschlagen, wo so etwas negativ bewertet werden kann. 

Die Prüfschritte 11.1.3.1b-11.1.3.1d, die sich alle auf Tabellen beziehen, ließen sich dagegen wohl gut in einem Prüfschritt zusammenfassen.

**Comment by johannesFischer84** [comment]
Gute Idee, der zusätzliche Prüfschritt. In so einen Sammelprüfschritt könnte man bei Bedarf auch Listen/Hervorhebungen bewerten, wozu ich etwas in #16 angemerkt hatte.

**Comment by sweckenmann** [comment]
Hm, aber im Web würde wir das nicht bei 1.3.1 bewerten, sondern bei 4.1.2 Name, Rolle, Wert, oder? Der Zustand wird nicht korrekt abgebildet? Oder ist das nicht vergleichbar?

**Comment by detlevhfischer** [comment]
> Hm, aber im Web würde wir das nicht bei 1.3.1 bewerten, sondern bei 4.1.2 Name, Rolle, Wert, oder
Ja, aber eher in Ermangelung eines geeigneten 1.3.1 Prüfschrittes. 4.1.2 ist ja eher für interaktive Widgets vorgesehen, die Zustände ändern können, wo diese Änderungen programmatisch ermittelbar sein sollen. Wenn aber überall Text einfach auf inert gesetzt ist, sehe ich das semantisch eher als 1.3.1 Problem.

**Comment by johannesFischer84** [comment]
> Hm, aber im Web würde wir das nicht bei 1.3.1 bewerten, sondern bei 4.1.2 Name, Rolle, Wert, oder? Der Zustand wird nicht korrekt abgebildet? Oder ist das nicht vergleichbar?

Auch im HTML würde ich Texte, Hinweise oder ähnliches unter 1.3.1 bewerten. Unter 4.1.2 fallen Bedienelemente, die WCAG benennt das als "user interface component" bzw. "single control for a distinct function". In 1.3.1 fallen dagegen Dokumentstrukturen, also die nicht-interaktiven Elemente.

**Comment by CarstenKaul** [comment]
Wird das vom TO beschriebene Problem nicht mit 11.5.2.10 abgedeckt? Das bezieht sich doch auf ALLE Textinhalte?!

**Comment by johannesFischer84** [comment]
> Wird das vom TO beschriebene Problem nicht mit 11.5.2.10 abgedeckt? Das bezieht sich doch auf ALLE Textinhalte?!

Auch hier ist wieder die Frage, ob man 11.5.2.10 und 1.3.1 für Text unterschiedlich prüfen könnte (siehe meine Anmerkung vorhin in #31). Ich denke, man benötigt auf jeden Fall in beiden Anforderungen eine Bewertungsmöglichkeit für Text.

**Comment by detlevhfischer** [comment]
Ich habe jetzt einen Prüfschritt entworfen: [1.3.1c Text programmatisch verfügbar](https://github.com/BIK-BITV/BIK-App-Test/blob/master/Pr%C3%BCfschritte/de/11.1.3.1c%20Text%20programmatisch%20verf%C3%BCgbar.adoc)

**Comment by detlevhfischer** [comment]
Ich habe außerdem den hier nicht benötigten Prüfschritt 11.1.3.1c "Zuordnung von Tabellenzellen" gelöscht.
